tailwind.config.js

```diff
@@ -110,10 +110,16 @@ module.exports = {
         700: "#3A6C92",
         800: "#082FD4",
       },
+      lightBlue: "#0964F4"
     },
     extend: {
       borderRadius: {
-        "2.5xl": "1.25rem",
+        DEFAULT: '5px',
```

Comment:
if we are overriding the default tw classes we can move it out from `extend`

Reply:
there are more style that we will need like `none`, `full` which we are inherited from default tw classes, thats why I have used extend here